### DISCUSSION TITLE
添加compare组件之后，尝试写对应的执行脚本compare_demo.py时运行失败

### DISCUSSION THREAD

**Discussion by Orphar** [secretflow/secretflow#1576] [Q&A] [answered]
文档中对于compare组件的定义时有
`meta =  VerticalTable()
input_table.meta.Unpack(meta)`
那么对应的compare_demo.py中是不是对应的要有
` sf_node_eval_param.inputs[0].meta.Pack(
       VerticalTable(
            schema=TableSchema(
                       ......
                ),
            line_count=-1,
         ),
    )
`
那么对应的VerticalTable要怎么写，根据报错信息来看，VerticalTable中是不允许有schema字段的
请问有没有类似于这种可以参考的例子

**Comment by lanyy9527** [comment, marked as answer]
可以参考：https://github.com/secretflow/secretflow/blob/v1.9.0b2/tests/component/preprocessing/filter/test_sample.py#L526
